frontend/pages/[station_category_slug]/[station_slug].tsx

```diff
@@ -1,106 +1,132 @@
+  station_slug,
+  default_station = false,
+}: {
+  stations_metadata: StationsMetadata;
+  station_category_slug: string;
+  station_slug: string;
+  default_station: boolean;
 }) {
-  const router = useRouter()
+  const router = useRouter();
   // TODO: Add a message when isLoading/isError are true
   const { stations, station_groups, isLoading, isError } = useStations({
     refreshInterval: 10000,
     initialStationsMetadata: stations_metadata,
   });
 
-  const random = (a: any[]) => a.find((_, i, ar) => Math.random() < 1 / (ar.length - i))
+  const random = (a: any[]) =>
+    a.find((_, i, ar) => Math.random() < 1 / (ar.length - i));
 
   const [selectedStationSlug, selectStationSlug] = useLocalStorageState(
     random(stations).slug,
     "SELECTED_STATION_SLUG",
   );
 
-  const [selectedStationGroupSlug, selectStationGroupSlug] = useLocalStorageState(
-    station_groups[0].slug,
-    "SELECTED_STATION_GROUP_SLUG",
-  );
+  const [selectedStationGroupSlug, selectStationGroupSlug] =
+    useLocalStorageState(station_groups[0].slug, "SELECTED_STATION_GROUP_SLUG");
   useEffect(() => {
-    if(default_station) {
-      router.push(`/${selectedStationGroupSlug}/${selectedStationSlug}`)
+    if (default_station) {
+      router.push(`/${selectedStationGroupSlug}/${selectedStationSlug}`);
       return;
     }
-  }, [])
+  }, []);
   useEffect(() => {
-    if(station_slug) {
-      selectStationSlug(station_slug)
+    if (station_slug) {
+      selectStationSlug(station_slug);
     }
-  }, [station_slug])
-
+  }, [station_slug]);
 
   // @ts-ignore
   const selectedStation: Station = stations.find(s => s.slug === station_slug);
 
-  const stationById = groupBy(stations, 'id');
+  const stationById = groupBy(stations, "id");
 
   // @ts-ignore
-  const selectedStationGroup: StationGroup = station_groups.find(s => s.slug === station_category_slug);
+  const selectedStationGroup: StationGroup = station_groups.find(
+    s => s.slug === station_category_slug,
+  );
 
-  const displayedStations = selectedStationGroup?.station_to_station_groups?.map((item) => {
-    return stationById[item.station_id];
-  }) || [];
+  const displayedStations =
+    selectedStationGroup?.station_to_station_groups?.map(item => {
+      return stationById[item.station_id];
+    }) || [];
 
   const pickARandomStation = () => {
     selectStationSlug(random(stations).slug);
-    router.push(`/${selectedStationGroupSlug}/${selectedStationSlug}`)
-  }
+    router.push(`/${selectedStationGroupSlug}/${selectedStationSlug}`);
+  };
 
   // TODO
   // TODO: add an option to search stations (eventually typing directly on the keyboard..)
 
-  const PageTitle = `Radiouri Crestine ${selectedStation && '- ' + selectedStation.title}`
+  const seo = {
+    title: `🟢 Radiouri Crestine ${
+      selectedStation && "- " + selectedStation.title
+    }`,
+    description: `Asculta ${selectedStation?.title} live · Lista de radiouri crestine · Radio Crestin Live`,
```

Comment:
Here the optimal description length should be between 155 and 160 characters.
https://moz.com/learn/seo/meta-description#:~:text=Optimal%20meta%20description%20length,between%2050%20and%20160%20characters.

For example we can add `station.description` to meta description
https://github.com/iosifnicolae2/radio-crestin.com/blob/cdfcf884e661d27f195c784ebd6c53a1178f9b23/frontend/components/StationInformation/StationInformation.tsx#L135


Also, it might be useful to add some emojis, for example [this](https://emojipedia.org/large-red-circle/) one - 🔴.